docs/CONTRIBUTING.md

```diff
@@ -11,178 +11,71 @@ Please see [SECURITY.md](SECURITY.md).
 ### Prerequisites
 
 - WordPress 6.7+
-- Gutenberg plugin (see [here](#custom-gutenberg-development))
-- PHP 8.2+
-- Node.js and npm
-- MySQL 8.0+ or MariaDB 10.6+ (for integration tests)
-
-### Project Initialization
-
-After cloning the repo, initialize the project by following these steps:
-
-1. **Install dependencies:**
-
-   ```bash
-   npm install
-   ```
-
-2. **Initialize husky:**
-
-   ```bash
-   npx husky init
-   ```
-
-   You may have to run this command to make the pre-commit hook operational. This will overwrite the `.husky/pre-commit` file, which you can revert after running the command.
+- Custom Gutenberg build in sibling directory (`../gutenberg`)
+  - This plugin currently requires a custom Gutenberg build ([latest release](https://github.com/Automattic/vip-go-mu-plugins-ext/tree/trunk/vip-integrations/gutenberg)). If you use our `wp-env`-based development environment, it expects this custom build to be present in a sibling directory named `gutenberg`.
```

Comment:
should we also include the branch where this is currently being worked on for those wanting to dive into and build it separately, or do we consider this enough?